locust/runners.py

```diff
@@ -367,6 +366,9 @@ def shape_worker(self) -> None:
                 #        of each load test shape stage.
                 self.start(user_count=user_count, spawn_rate=spawn_rate, user_classes=user_classes)
                 self.shape_last_tick = current_tick
+                shape_adjustment_time_ms = time.time() - shape_adjustment_start
+                if shape_adjustment_time_ms < 1:
```

Comment:
`<1` feels strange :)

Can you maybe change it to not use an if statement at all and instead do:

gevent.sleep(Math.max(0, 1 - shape_adjustment_time_ms))

Also, the build seems to be breaking...

Reply:
Yep, just noticed the test, sorry! I'll fix that. the `max(..., 0)` seems good, will adjust the commit then :)

Reply:
I haven't noticed that overriding `current_tick` leads to unwanted behavior. Fix was commited